dashboard/config/locales/dsls.es-MX.yml

```diff
@@ -2838,6 +2942,377 @@ es-MX:
+      OPD-K5 Culture:
+        markdown: |+
+          # Mejores prácticas > Cultura en el aula
+
+          ¿Cómo podemos promover una cultura saludable en el aula para nuestras clases de computación? Primero, podemos asegurarnos de que todos en el aula, incluidos nosotros mismos, comprendamos las formas en que podemos actuar como informáticos y qué tener en cuenta. Esto mantiene a todos en la misma sintonía y facilita tu trabajo en el aula.
+
+          <img src="https://images.code.org/b5d9ae18a882a4359f096b9c6ade135e-image-1434398703448.jpg" width="450px" style="margin-right:45px; border-radius:10px;">
+
+          ## Formas de actuar como un informático:
+
+          - **Sé curioso:** Cuando eres curioso, haces cosas como hurgar en diferentes menús, ejecutar fragmentos de código para ver qué sucede o verificar los efectos de diferentes configuraciones. La curiosidad, a menudo, lleva a poder responder a tus propias preguntas.
+          - **Haz preguntas:** Colabora con otros una vez que hayas intentado resolver un problema tú mismo.
+          - **Sé persistente:** Ten confianza para abordar problemas complejos abiertos y quédate con ellos. Esto se vuelve más fácil una vez que has escrito algunos programas exitosos y sabes lo bien que se siente superar un problema de programación difícil.
+          - **Divide los problemas grandes en problemas más pequeños y manejables:** Esto se llama **descomposición**.
+          - **Lee tu código: ** La computadora sigue tus instrucciones exactamente. El rastreo a través del código paso a paso (como si fuera la computadora que procesa las instrucciones) te ayudará a encontrar errores en tu lógica o pasos faltantes.
+          - **No te sientas mal por cometer errores:** Los errores no son fallas; solo te acercan un paso más a la creación de la solución o producto final adecuado.
+          - **Aléjate de tu pantalla:** Esto puede darte un descanso muy necesario y, a menudo, puedes volver a un problema con una nueva perspectiva.
+
+          ## Cosas por tener en cuenta al aprender informática:
+
+          - El mundo de la programación de computadoras se mueve tan rápido que realmente no hay profesores. **Todos somos aprendices** en diferentes etapas, pero todavía somos siempre aprendices.
+          - La programación se vuelve más cómoda cada vez que vuelves a ella. **No vas a ser un experto la primera vez.**
+          - No hay forma de evitarlo, el acto de programar y **aprender a programar lleva tiempo, ¡pero cualquiera puede hacerlo!**
+          - ¡Puedes compartir **tu propia experiencia auténtica aprendiendo a codificar** con tus estudiantes!
+
+          > ## Reflexionar:
+          > ¿Cómo planeas crear una cultura positiva en el aula cuando enseñas informática? [Tuitea tus pensamientos] (https://twitter.com/codeorg) y asegúrate de etiquetar @codeorg en tu tweet para que podamos responder.
```

Comment:
@castro-jorge looks like a ton of these translations are putting a space in the middle of the link syntax